kernel/aster-nix/src/fs/fs_resolver.rs

```diff
@@ -318,6 +318,29 @@ impl FsResolver {
 
         Ok((dir_dentry, base_name))
     }
+
+    /// The method used to create a new file using pathname.
+    ///
+    /// For example, mkdir, mknod, link, and symlink all need to create
+    /// new file and all need to perform unique processing on the last
+    /// component of the path name. path_create is used to provide a unified
+    /// method for pathname lookup and error handling.
+    pub fn path_create(
```

Comment:
Simply calling this method `create` to comply with Rust naming convention, which requires every method name to start with a verb (unless the method is a getter or setter).

Reply:
Thank you for pointing this out, but isn't it a bad idea to just name it `create`? Because create is the purpose of using pathname. In this method, only pathname lookup and some error handling are performed, and no actual so-called create operation is performed.

Reply:
Yes, you are right: `create` would make a bad name. How about `lookup_dir_and_new_basename`?

Reply:
I think it can be temporarily called `lookup_dir_and_new_basename`, but as I said about the classification of system calls, `lookup_dir_and_new_basename` is provided for `mkdir`, `mknod`, `link`, `symlink`. Other categories also need to define similar methods and have a suitable name. I think we can standardize the naming of these methods later.